Sources/AblyChat/Version.swift

```diff
@@ -0,0 +1,16 @@
+import Ably
+
+// Update this when you release a new version
+
+// Version information
+public let VERSION = "0.1.0"
```

Comment:
These capitalised constants aren’t very Swift-like; they could just be `let`s` with camel-cased names.

Reply:
fixed, also added a todo